diff --git a/dist/index.cjs b/dist/index.cjs
index ef59f264e4d56dc8d9055054828ed67877a20716..4e3a02bd12a771d730f75a4473dcdcc03f48fea7 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -1450,7 +1450,7 @@ function PitchSequence({
     }))
   ] : rows;
   const handleSelect = onSelect ? (i, row) => {
-    if (row._placeholder) return;
+    if (row?._placeholder) return;
     onSelect(i, row);
   } : void 0;
   return /* @__PURE__ */ (0, import_jsx_runtime16.jsx)(
@@ -1464,7 +1464,7 @@ function PitchSequence({
       onSelect: handleSelect,
       focused: active,
       onRowHover: (i, row) => {
-        if (row._placeholder) return;
+        if (row?._placeholder) return;
         report(i);
       },
       style
diff --git a/dist/index.js b/dist/index.js
index d1bcba41172990a37a2d1bdb5304531030fff6dc..91b217fbb85fc7d2b379ff5ef38cd816adca96cf 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1396,7 +1396,7 @@ function PitchSequence({
     }))
   ] : rows;
   const handleSelect = onSelect ? (i, row) => {
-    if (row._placeholder) return;
+    if (row?._placeholder) return;
     onSelect(i, row);
   } : void 0;
   return /* @__PURE__ */ jsx16(
@@ -1410,7 +1410,7 @@ function PitchSequence({
       onSelect: handleSelect,
       focused: active,
       onRowHover: (i, row) => {
-        if (row._placeholder) return;
+        if (row?._placeholder) return;
         report(i);
       },
       style
